AboutDoc/Documentation_About.rst

```diff
@@ -0,0 +1,250 @@
+
+Software 
+
+- Notepad++ ReST template:	https://github.com/steenhulthin/reStructuredText_NPP
+- Linux: 
+
+File naming and directory structure
+-----------------------------------
+
+| Files are stored in \docs
+| File names may not contain spaces. Spaces in filenames cause problems with Sphinx. 
+| For multi-word filenames, ReST documentation filenames are separated by an underscore ( _ ).
+| Image filenames are separated by a dash ( - ). 
+| Images are named after the file they belong to, and sorted in the doc\\images subdirectory. 
+
+.. NOTE: Or maybe not, see Images
+
+Translations go into their own directory. 
+
+File structure
+--------------
+
+Every file starts with 5 hidden segments (See `template <Template.rst>`_ )
+
+- Copyright notice
+- Openshot description
+- Openshot disclaimer
+- License notice
+- Referral anchor for the title
+
+Followed by the title with double underlining (with == )
+Then a short description of what the chapter will be about, and why the reader should care (or skip). 
+Subtitles are single underlined ( -- )
+
+Sentences should be written one sentence per line, and do not need to end on a space.
+The markup language then flows them all together into paragraphs when it generates the formatted docs.
+You can also break at other logical points, like after a comma in a longer sentence,
+or before starting an inline markup command.
+It is a guideline, not a rule.
+This tends to be a pretty good fit for any sort of written prose, when it is in a markup language like ReST and managed in a version control system.
+
+There are four reasons for this:
+
+- Writing that way, there is no worrying about line length or when to wrap. 
+- It encourages shorter, simpler sentences which is a good thing when writing docs. 
+- The diffs when changes are submitted also tend to be more readable and focused. 
+- Lines are easier to translate and less likely to be changed. 
+
+Comments for why things are documented a certain way can be hidden after a double dot and start with "NOTE: ". 
+They may contain a link to a relevant issue in the tracker for more information. 
+
+Translation
+-----------
+Translation files are generated and managed by Sphinx.
+If the images are not translated, they will default back to the original.
+Filenames do not get translated.
+There may be translation notes hidden in the documentation, blocked out with \.. TRANSLATION NOTE: 
+
+Files for translation will be hosted at `Launchpad <https://translations.launchpad.net/openshot/2.0/+translations>`_.
+
+When translating numbers referencing a screenshot in non-western languages, please make sure to update the screenshot too. 
+If available, images of the translation should be saved in their subdirectory *(to be decided)* 
+
+.. TODO: Add subdirectory
+
+.. TRANSLATION NOTE: After translating tables, make sure that the underlining of table rows stay the same length as the new words. 
```

Comment:
Nope to all of this, too. It should probably go. :disappointed:

Reply:
>Drawing a distinction between "the online HTML and the offline manual" is a bit dubious here, since they're exactly the same HTML. A copy of what is otherwise the "offline manual" gets uploaded to the openshot.org site after it's generated by the build servers. (So, certainly the sentence as-is isn't wrong... other than the capitalization.)

Hey, that is on you, for explaining Sphinx too well. 

"Plus, WYSIWYG to some extent implies a particular layout, whereas docs can be generated from reST using a variety of styles and formats, so the same docs will look very different depending how it's processed."
If it _could_, I thought it _would_. So I presumed that the offline doc would be PDF. (And possibly Epub, Winhelp/CHM/MShelp2, RTF, Writer and whatnot versions could exist.)

If they do not exist, then that line can indeed be shortened. 

>> Or in *this* reddit thread. 
> Ah. Maybe remove the previous line, and jut make the TODO about adding Reddit info?

**It's a proposal that I did not feel up to me to decide. **
**It also depends on if topics on Reddit can be sticky to the top, how often you expect changes to the Help be needed, and if you have someone to regularly check it. **
I thought it could be a low-level way for people to make small suggestions. But if it is on top it may get abused, and if it is not on top it will be unable to find. 
Since the meta-documentation is on github, it is probably fair to asume that the readers have an account. But if there are other places that suggestions for the docs are made, they should be listed so you can check them to see if something needs to be added. 

>I don't know that it's really worth belaboring all the details here. 

(Re license info) I agree that it could be phrased shorter, but I was intentionally *going* for completeness of the details. 
In five years the docs may need another large overhaul and someone is bound to ask; 'why is this not under CC license?' (Like I did). And you may not be here to explain. 

>The 'question' tag is actually auto-applied to any issue opened using the Question

Every 'question' is a question, but not all questions are the same. 
Questions that get asked a lot should be in the FAQ, even (especially) if the answere is no, the asker does not stay around or it is already in the docs. 

Questions with a good/complex answer may (partially) be used to update the docs or be in the FAQ, even if they are also an enhancement or bug. 

So **yes, I am proposing the creation of a FAQ-tag and more rigorous use of the Doc-tag.** (Every time a new feature is added, it should be documented or screenshots may need updating. 
For example, I am waiting for https://github.com/OpenShot/openshot-qt/pull/3346 to make it into the dailys so the framenumber will be included in the new screenshots.

Reddit also seems to support tags, though I have not figured out how yet. If you have a regular Reddit-watcher, (s)he may want to add tag too. 

> Again, nix all mentions of translation please.

Yes, you mentioned that at the end of the other topic, was leaving it in for completion. (And maybe someone has an alternative)
Disappointing that it won't work in Sphinx. However, shorter lines may remain relevant if you process the Rest source files in other translation programs. OTOH, translating entire paragraphs can have benefits as some languages prefer longer sentences. 
So I propose to hide it, and leave it till a process for translation is decided upon.

Reply:
You can reply to individual comments directly, in context, BTW. Instead of combining them all together. It helps a lot, in keeping track of the various points raised; each of the individual conversations can be marked "Resolved" when that item has been dealt with.